This patch makes sure that the xtests programs don't get build when running
'make all', as they might fail to build (for instance if GLIBC 2.3 is used).

Note that the tests are not executed by default at make check because they
are anyway broken.
Index: Linux-PAM-0.99.9.0/xtests/Makefile.am
===================================================================
--- Linux-PAM-0.99.9.0.orig/xtests/Makefile.am
+++ Linux-PAM-0.99.9.0/xtests/Makefile.am
@@ -29,7 +29,7 @@ XTESTS = tst-pam_dispatch1 tst-pam_dispa
 	tst-pam_access4 tst-pam_limits1 tst-pam_succeed_if1 \
 	tst-pam_group1
 
-noinst_PROGRAMS = $(XTESTS)
+check_PROGRAMS = $(XTESTS)
 
 xtests: $(XTESTS) run-xtests.sh
 	"$(srcdir)"/run-xtests.sh "$(srcdir)" ${XTESTS}
